### DISCUSSION TITLE
new users

### DISCUSSION THREAD

**Discussion by jpv-1** [freeplane/freeplane#1517] [Ideas]
Hi. To facilitate access for new users, would it be possible to simplify the creation of new documents:
1- templates: display the model on mouseover, without needing to click and then return to the list
2- possibly propose models (templates) as currently but also some diagrams already started (5w, fishbone)
3- possibly allow starting the diagram directly without going through the model proposal in the center (display the model proposals in a window on the side with the possibility of not taking them into account)
THANKS

**Comment by Mybabies2023GAC** [comment]
I am using freeplane to map root cause analysis. I am looking for a feature that can help me document evidence for each bubble, I'd like to know what is available. Thank you.

**Reply by euu2021** [reply]
Searching for "root cause analysis diagram" in Google Images, I see that the Fishbone diagram is the most popular kind of diagram for this purpose. As discussed [here](https://github.com/freeplane/freeplane/discussions/1083#discussioncomment-7400711), it's not easy to create this kind of diagram on FreePlane.

I never used this kind of diagram, but I really don't see the advantage of using it over using a tree diagram (the usual Freeplane layout). For example, in the second image of [this post](https://github.com/freeplane/freeplane/discussions/1083#discussioncomment-7406558), I displayed the Fishbone diagram as a tree diagram.
